packages/palette/src/elements/BaseTabs/BaseTab.story.tsx

```diff
@@ -0,0 +1,182 @@
+import { action } from "@storybook/addon-actions"
+import { storiesOf } from "@storybook/react"
+import React from "react"
+import { BaseTab, BaseTabs } from "."
+import { ChevronIcon } from "../../svgs"
+import { Box } from "../Box"
+import { Clickable } from "../Clickable"
+
+// Fake `RouterLink`
+const RouterLink: React.FC<{ to: string }> = ({ to, children, ...rest }) => {
+  return (
+    <a href={to} {...rest}>
+      {children}
+    </a>
+  )
+}
+
+storiesOf("Components/BaseTabs", module)
+  .add("Default", () => {
+    return (
+      <BaseTabs m={3}>
+        <BaseTab>First</BaseTab>
+        <BaseTab>Second</BaseTab>
+        <BaseTab active>Active</BaseTab>
+        <BaseTab>Last</BaseTab>
+      </BaseTabs>
```

Comment:
I think the answer is `yes` but is the choice to use names like `BaseTabs` and `BaseTab` because, at some higher and more product-dev centric everyday level, we'll see these things implemented and used like 

```tsx
<Tabs>
  <Tab>...</Tab>
</Tabs>
```
?

Reply:
Extended yes; both in Force and used to build out the client-side tabs here. But also pragmatically. I'd like them to be called `<Tablist>` `<Tab>` but wouldn't be able to do that without drastically altering the API (perhaps worth doing if I'm going to cut a major release anyway).

Reply:
> Perhaps worth doing if I'm going to cut a major release anyway

Yup, why not? That naming seems a little better, and if things are gonna be updated then might as well just rip it out and then delete old tab component from palette. 

(Either way, this is non-blocking)